aday2dream/src/main/kotlin/com/aday2dream/aday2dream/service/JwtService.kt

```diff
@@ -14,15 +15,16 @@ import java.util.Date
 
 object JwtService {
 
-    private val SECRET_KEY: Key = Keys.secretKeyFor(SignatureAlgorithm.HS256)
-    private const val MINUTES: Long = 60
+    private var SECRET_KEY: Key = Keys.secretKeyFor(SignatureAlgorithm.HS256)
+    @Value("\${jwt.minutes}")
+    private val minutes: Long = 60
```

Comment:
if you inject it via the application.properties, i dont think you need to give it a value here, because you over-write the value from application properties!

Reply:
I think this is because you dont need the \ in the path to the value
means that basically, you are not injecting correctly the value from app properties